Description: Upstream changes introduced in version 1.2.3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 mumble (1.2.3) maverick; urgency=low
 .
   * PPA Upload of 1.2.3 for Ubuntu maverick
 .
 The person named in the Author field signed this changelog entry.
Author: Zhenkai Zhu <zhenkai@cs.ucla.edu>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ mumble-1.2.3/CHANGES
@@ -0,0 +1 @@
+test
